index.bs

```diff
@@ -4687,15 +4687,45 @@ Note: If an error name is not listed here, please file a bug as indicated at the
         <tbody>
             <tr>
                 <td>
-                    "<dfn id="invalidaccesserror" exception export><code>InvalidAccessError</code></dfn>"<br>
-                    <dfn id="dom-domexception-invalid_access_err" for="DOMException" const export><code>INVALID_ACCESS_ERR</code></dfn>&nbsp;(15)
+                    "<dfn id="domstringsizeerror" exception><code>DOMStringSizeError</code></dfn>"<br>
+                    <dfn id="dom-domexception-domstring_size_err" for="DOMException" const><code>DOMSTRING_SIZE_ERR</code></dfn>&nbsp;(2)
+                </td>
+                <td>Use <emu-val>RangeError</emu-val> instead.</td>
+            </tr>
+            <tr>
+                <td>
+                    "<dfn id="nodataallowederror" exception><code>NoDataAllowedError</code></dfn>"<br>
+                    <dfn id="dom-domexception-no_data_allowed_err" for="DOMException" const><code>NO_DATA_ALLOWED_ERR</code></dfn>&nbsp;(6)
+                </td>
+                <td>—</td>
```

Comment:
I was going to suggest giving alternatives, but I don't know what these were used for once upon a time, so I'd just be guessing. So leaving with --- seems fine.